### PR TITLE
OPSDIR: make it explicit that we're not talking about active measurement

### PR DESCRIPTION
**Pull request by britram**

### CODE REVIEW

**Review by LPardue — APPROVED** [review]

**Comment by britram** [comment]
hi @LPardue @acmacm @mirjak PTAnotherL, reviewed the thread and I do think an additional 9065 ref here would be helpful.

File: draft-ietf-quic-manageability.md [hunk 1]
```diff
@@ -107,7 +107,13 @@ requirements, for example, would be impermissible on other networks without
...
 recommendations as to which practices should or should not be applied; for each
 practice, it describes what is and is not possible with the QUIC transport
-protocol as defined.
+protocol as defined. 
+
+This document focuses solely on network management practices that interact
+with traffic on the wire. Replacement of troubleshooting based on observation
+with active measurement techniques, for example, is therefore out of scope.
+A more generalized treatment of network management operations on encrypted
+transports is given in {{?RFC9065}}.
```
Comment (britram) [inline draft-ietf-quic-manageability.md:116]:
```suggestion
protocol as defined.

This document focuses solely on network management practices that interact
with traffic on the wire. Replacement of troubleshooting based on observation
with active measurement techniques, for example, is therefore out of scope.
A more generalized treatment of network management operations on encrypted
transports is given in {{?RFC9065}}.
```

File: draft-ietf-quic-manageability.md [hunk 2]
```diff
@@ -109,6 +109,12 @@ recommendations as to which practices should or should not be applied; for each
 practice, it describes what is and is not possible with the QUIC transport
 protocol as defined.
 
+This document focuses solely on network management practices that interact
```
Comment (acmacm) [inline draft-ietf-quic-manageability.md:112]: s/interact/observe/   better?   Otherwise ok.

Comment (britram) [reply draft-ietf-quic-manageability.md:112]: Good point. Making this change. thanks!

**Pull request merged**